Formula/stanford-corenlp.rb

```diff
@@ -18,7 +16,7 @@ def install
 
   test do
     (testpath/"test.txt").write("Stanford is a university, founded in 1891.")
-    system "#{bin}/corenlp.sh", "-annotators tokenize,ssplit,pos", "-file test.txt"
-    assert_predicate (testpath/"test.txt.xml"), :exist?
+    system "#{bin}/corenlp.sh", "-annotators tokenize,ssplit,pos", "-file test.txt", "-outputFormat json"
```

Comment:
Explicitly adding output format for the file, as 4.2.0 requires this option for file output formats.